### DISCUSSION TITLE
Icon sizing fails after installing 2024-07 Cumulative Update Preview for Windows 11 Version 23H2 for x64-based Systems (KB5040527)

### DISCUSSION THREAD

**Discussion by tepollack** [ramensoftware/windhawk-mods#815] [Q&A]
Icon sizing fails after installing 2024-07 Cumulative Update Preview for Windows 11 Version 23H2 for x64-based Systems (KB5040527) 

System Information: 
Edition Windows 11 Pro
Version 23H2
Device name Asus-Z690
Processor 13th Gen Intel(R) Core(TM) i5-13600K 3.50 GHz
Installed RAM 32.0 GB (31.7 GB usable)
Device ID 9912576B-A115-4DOB-9E6D-C42AB1714476
Product ID 00330-80000-OOOOO-AA937
System type 64-bit operating system, x64-based processor
Pen and touch No pen or touch input is available for this display

**Comment by m417z** [comment]
Update KB5040527, which was released two days ago, introduced a change that made it incompatible with the mod. Should be fixed now in version 1.2.9.

See also: https://github.com/ramensoftware/windhawk-mods/issues/809.

**Comment by tepollack** [comment]
Thanks.  If only all customer service was this efficient. Time to send
another donation.

Tom

On Sat, Jul 27, 2024 at 8:22 AM Michael Maltsev ***@***.***>
wrote:

> Update KB5040527, which was released two days ago, introduced a change
> that made it incompatible with the mod. Should be fixed now in version
> 1.2.9.
>
> See also: #809 <https://github.com/ramensoftware/windhawk-mods/issues/809>
> .
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ramensoftware/windhawk-mods/discussions/815#discussioncomment-10167128>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BJKBQYDLAWOWT73DT7MZIYLZOOGI7AVCNFSM6AAAAABLR2XBLGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMJWG4YTEOA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> .com>
>


-- 
Tom Pollack

**Reply by m417z** [reply]
Thank you for the support Tom, I appreciate it!
